### ISSUE TITLE
Great person spending

### ISSUE THREAD

**Issue by Der-Zorn-Gottes** [LoneGazebo/Community-Patch-DLL#1078]
I have Splendor (+50 culture + 50 GA points when you spend a GP), and Ceremonial Burial (+50 culture and +25 GA points when you spend a GP).
I am in the Rennaisance, so the era modifier is 3. I play marathon, so the speed modifier is 3.
The expected yields are +450 Culture, +450 Faith, +450 Culture, +225 Faith.

The actual yields are:
+ 450 Culture, +750 Faith, +450 Culture, +225 Faith.

So one Faith yield is too high.
I suppose something is strange with the era scaling, because 750 is not divisible by 9.

**Comment by Der-Zorn-Gottes** [comment]
Sorry, it is actually sulture that gives 750.

**Comment by LoneGazebo** [comment]
sulture? So is this a bug, or no?

**Comment by Der-Zorn-Gottes** [comment]
Kind of.
I spent some time thinking about it, and remembered I also have the Mausoleum, that gives 30 culture upon spending.
I checked in modern times:
375 Faith
750 Culture
750 Faith
1200 Culture
I schould get 750 Culture from CB, and 450 Culture from Mausoleum - it fits.
Could it be that the game just adds up two yields from completely different sources, and displays the sum instead of two single values? If so, why does it add a wonder effect with a Social policy effect but displays the effect from a belief alone?
My theory doesn`t quite hold up to the values I posted in my initial report, it should be 720 Culture there. However, it is completely possible that I it was actually 720, and I just misread. These numbers vanish quickly if you are trying to catch 4 values.
In short:
I think what happns is that the game displays yields form Mausoleum and Ceremonial Burial together.
No bug, but misleading.

**Comment by LoneGazebo** [comment]
It does because they both proc in the same function element. So all is correct
